crowbot/apiai_connection/crowbot_chat.py

```diff
@@ -202,8 +202,16 @@ def ask_apiai(text):
     elif response["result"]["metadata"]["intentName"] == "Default Fallback Intent":
         #legge til spørsmål i Questions modell
         question = response["result"]["resolvedQuery"]
-        words = re.compile('\w+').findall(question)
+        # om spørsmålet slutter med '?', fjern dette
+        if question.strip().endswith('?'):
+            question = question.strip()[:-1]
```

Comment:
Her kjøres `.strip()` to ganger på samme string - det hadde vært bedre å gjøre det én gang før `if`-en, så slipper vi å gjøre akkurat samme jobb to ganger.